sdcm/argus_results.py

```diff
@@ -171,51 +171,86 @@ def submit_results_to_argus(argus_client: ArgusClient, result_table: GenericResu
             raise
 
 
-def send_result_to_argus(argus_client: ArgusClient, workload: str, name: str, description: str, cycle: int, result: dict,
-                         start_time: float = 0, error_thresholds: dict = None):
-    result_table = workload_to_table[workload]()
+def send_result_to_argus(argus_client: ArgusClient, workload: str, name: str, description: str,  # noqa: PLR0914
```

Comment:
we do too much in this function...
How about keeping `send_result_to_argus` as close as possible to original (make `cycle` a `str`) and introduce new function for sending summary (or function to prepare summary as result dict and reuse this func)?
at least, add some docs how it works.

Reply:
I'm not sure splitting this would make it any easier, and I don't want to read the HDR data multiple times
as long we have enough unittest to cover this complexity, I think it's reasonable.

Reply:
> we do too much in this function... How about keeping `send_result_to_argus` as close as possible to original (make `cycle` a `str`) 

Parameters were not changed. The `cycle` was `int` and stayed `int`.
It is kept as close as possible to the original with changes needed for new scenario - multiple stress commands.

> and introduce new function for sending summary (or function to prepare summary as result dict and reuse this func)? 

Any split will lead to a redundancy in the data processing.

> at least, add some docs how it works.

Ok, will add docstring.